server/db_component.cpp

```diff
@@ -333,36 +336,70 @@ bool DB::Initialized()
 	return connection != nullptr;
 }
 
-string DB::PathFromID(unsigned int id, string * title, string * artist)
+string DB::PathFromID(unsigned int id, string * title, string * artist, string nspace)
```

Comment:
nspace has a default value - therefore it must be at the end. In c/c++ once you go default, you can't go back.